tests/suites/test_suite_psa_crypto_pake.function

```diff
@@ -535,7 +536,7 @@ exit:
  * END_DEPENDENCIES
  */
 
-/* BEGIN_CASE depends_on:PSA_WANT_ALG_JPAKE */
+/* BEGIN_CASE depends_on:PSA_WANT_ALG_JPAKE:MBEDTLS_HAS_ALG_MD5_VIA_MD_OR_PSA */
```

Comment:
I do not see any dependency on MD5 or any hash in the code of this function thus I am not sure what this is for.

Reply:
ok but then I think it rather belongs to the .data file, doesn't it?

Reply:
As far as I remember in previous comments we were moving SHA-256 dependency form `.data` to `.function` file. My understanding is that it apples to all test cases as `ecjpake.c` requires SHA-256 via MD or PSA.

Reply:
We moved some dependencies from .data to .function because the dependency was in the code, not in the data: the code was explicitly referring to SHA-256. But here this is not the case I think. The test data are referring to SHA-256 thus the dependency in that case should be in the .data file.